import template from './admin.html';
import './admin.styl';

const adminComponent = {
    template,
    bindings: {

    },
    controller: /* @ngInject */ 
    class AdminController {
        static get $inject() {
            return ['$log', '$timeout', '$scope', 'FormApi', 'adminService'];
        }
        constructor($log, $timeout, $scope, FormApi, adminService) {
            this.$log = $log;
            this.$timeout = $timeout;
            this.$scope = $scope;
            this.FormApi = FormApi;
            this.adminService = adminService;
        }
        $onInit() {
            this.selectAllAbs = [];
            this.getAbstracts();
            // this.sortByOptions = Object.keys(this.getAbstracts()[0]);
            this.sortByOptions = {
                name: ["Title", "Author", "Field"],
                value: ["title", "fileId", "field"]
            };
            this.sortBy = '';
            this.displaySetting = {
                showAffil: false,
                showKeywords: false,
                showAbsContent: false
            };
            this.selectedAbs = [];
            
            // this.adminService.getData();
            // this.adminService.storeData({'change':'value'});
        }
        $onDestroy(){
            // this.adminService.cleanup();
        }
        test(input) {
            if (input.canGiveATalk === "true") {
                return true
            } else { return false }
            // console.log(typeof input.canGiveATalk);
            // return Boolean(input.canGiveATalk);
        }
        getAbstracts() {
            // return [{"_id":"57f168e252f9a10003cd79b0","email":"asdscott@gmail.com","title":"Eg5 Inhibitors Have Contrasting Effects on Microtubule Stability and Metaphase Spindle Integrity","field":"Bioengineering","authors":[{"name":"Geng-Yuan Chen","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Department of Bioengineering, Pennsylvania State University, University Park, PA"],"validAuthor":true},{"name":"You-Jung Kang","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Bioengineering, Pennsylvania State University, University Park, PA"],"validAuthor":true},{"name":"A. Sophia Gayek","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN"],"validAuthor":true},{"name":"Wiphu Youyen","role":"General","affiliationSup":[3],"affiliationOfAuthor":["Department of Physics, Worcester Polytechnic Institute, Worcester, MA"],"validAuthor":true},{"name":"Erkan Tüzel","role":"General","affiliationSup":[3],"affiliationOfAuthor":["Department of Physics, Worcester Polytechnic Institute, Worcester, MA"],"validAuthor":true},{"name":"Ryoma Ohi","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN"],"validAuthor":true},{"name":"William O. Hancock","role":"Corresponding","affiliationSup":[1,null],"affiliationOfAuthor":["Department of Bioengineering, Pennsylvania State University, University Park, PA"],"validAuthor":true}],"affiliations":["Department of Bioengineering, Pennsylvania State University, University Park, PA","Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN","Department of Physics, Worcester Polytechnic Institute, Worcester, MA"],"useAffiliationSup":true,"keywords":"kinesin, microtubule, mitosis, single-molecule enzyme kineitcs","absContent":"The goal was to investigate the mechanisms underlying different Eg5 inhibitors and to assess their effects on Eg5-mediated microtubules (MTs) stabilization.  Antimitotic Eg5 inhibitors were grouped into those that bind Loop-5 and produce an ADP-like state (monastrol, STLC, ispinesib and filanesib) and those that produce a rigor-like state (BRD9876).  To understand their contrasting effects on Eg5 mechanics, inhibitors were analyzed in MT motility assays consisting of different ratios of kinesin-1 and Eg5 motors, in which Eg5 “braking” dominates motility. Loop-5 inhibitors increased MT motility, consistent with the compounds inducing a weak-binding Eg5, whereas BRD9876 slowed gliding, consistent with Eg5 being locked in a \"rigor\". Quantitative stall force analysis indicates that L5 inhibitors reduce Eg5 detachment force whereas BRD9876 generates an Eg5 species with enhanced detachment force. Biochemical and single-molecule assays demonstrated that BRD9876 acts as an ATP/ADP-competitive inhibitor with a KI of 4 nM. Consistent with its recently reported MT polymerase activity _(Chen and Hancock, Nat. Comm. 2015:8160)_ and its dominant two-head-bound strong-binding population _(Chen et al., JBC 2016:M116)_, Eg5 was shown to stabilize MTs against depolymerization. This stabilization was eliminated in monastrol but was enhanced by BRD9876, consistent with strong binding of Eg5 to MTs enhancing MT stability. Finally, in metaphase-arrested RPE-1 cells, STLC promoted spindle collapse, whereas BRD9876 did not. Thus, different Eg5 inhibitors inhibit spindle formation through contrasting mechanisms and rigor inhibitors are predicted to stabilize MTs in cells.","createdAt":"2016-10-02T19:30:10.907Z","submittedAt":["2016-10-02T20:06:57.840Z"],"canGiveATalk":"true"},{"_id":"57fe81d6325e1700036473bc","email":"crjray@gmail.com","title":"Structural Studies and Structure-Guided Engineerings of Epimerization Domains to Reprogram Antibiotic Biosynthetic Pathways","field":"Biochemistry","authors":[{"name":"Wei-Hung Chen","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Department of Chemistry, University of Florida, Gainesville, FL"],"validAuthor":true},{"name":"Naga Sandhya Guntaka","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Chemistry, University of Florida, Gainesville, FL"],"validAuthor":true},{"name":"Steven D. Bruner","role":"Corresponding","affiliationSup":[1],"affiliationOfAuthor":["Department of Chemistry, University of Florida, Gainesville, FL"],"validAuthor":true}],"affiliations":["Department of Chemistry, University of Florida, Gainesville, FL"],"useAffiliationSup":false,"keywords":"nonribosomal peptide synthetase, epimerase, and antibiotics biosynthesis pathway","absContent":"Nonribosomal peptide synthetases are large, complex multi-domain enzymes responsible for the biosynthesis of a wide range of peptidyl natural products. The addition of further chemical modifications contribute significantly to the structural diversity of nonribosomal peptides, adding functional diversity and _in vivo_ stability. Notably, the epimerization (E) domain in NRPS assembly lines is a common and important modification for bioactive nonribosomal peptides.\n\nDespite the previous biochemical and structural studies, several questions related to E domain substrate recognition and the detailed chemical mechanism remain to be addressed. In addition, general questions related to NRPS domain orientation and the flux of Ppant-tethered intermediates through the megasynthetase are unclear. The issues of structure and mechanism have direct relevance to engineer NRPSs through domain manipulations and rearrangements to produce novel compounds, although studies have shown success, generally applicable approaches are not known. \n\nIn this study, we present the structure of the epimerization domain/peptidyl carrier protein didomain construct from the first module of the gramicidin S synthetase. From the structures, novel details related to the interactions between catalytic residues/the Ppant prosthetic group and the interface region within linker-domain and domain-domain are apparent. Additionally, the information revealed within structures provides insights into domain-domain recognition, substrate delivery during the assembly line process and guides us to carry out a structural based engineering resulting in a relaxed substrate specificity. Finally, the structures supply information into for the structural organization of homologous condensation domains, canonical players in all synthetase modules.","createdAt":"2016-10-12T17:52:33.270Z","submittedAt":["2016-10-12T18:32:56.150Z","2016-10-12T18:34:54.459Z","2016-10-12T18:36:59.061Z"],"canGiveATalk":"true","updatedAt":"2016-10-12T18:36:59.063Z"},{"_id":"57fffcbd6856f40003948d10","email":"yenshuoting@gmail.com","title":"CreLite: A Red Light-Inducible Cre Tool for Mouse and Zebrafish","field":"Developmental biology","authors":[{"name":"Shuo-Ting Yen","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Program in Developmental Biology, Baylor College of Medicine, Houston, TX"],"validAuthor":true},{"name":"Jian-Min Deng","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Genetics, University of Texas MD Anderson Cancer Center, Houston, TX"],"validAuthor":true},{"name":"George T. Einsenhoffer","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Genetics, University of Texas MD Anderson Cancer Center, Houston, TX"],"validAuthor":true},{"name":"Richard R. Behringer","role":"Corresponding","affiliationSup":[2],"affiliationOfAuthor":["Department of Genetics, University of Texas MD Anderson Cancer Center, Houston, TX"],"validAuthor":true}],"affiliations":["Program in Developmental Biology, Baylor College of Medicine, Houston, TX","Department of Genetics, University of Texas MD Anderson Cancer Center, Houston, TX"],"useAffiliationSup":true,"keywords":"Optogenetics, Cre recombinase, Phytochrome B, phycocyanobilin","absContent":"Precise manipulation of gene expression with spatial and temporal control has been an important goal for developmental biology studies. Currently, the Cre-loxP binary system allows gene expression control using known ubiquitous or tissue-specific regulatory elements, which limits the possibilities for the control of gene expression at desired times and places. Recently, light-inducible proteins have become a powerful tool for genetic manipulations and confer precise control of various biological systems. A blue light-inducible Cre system has been engineered and applied in fly and a cell culture system. However, the low tissue penetrance and high photo-toxicity of blue light may hinder further applications for live, thick tissue or animals. Here, we present an engineered red light-inducible Cre system, CreLite, for potential use in both mouse cell culture and zebrafish embryos. The red light-inducible Cre system consists of two fusion proteins, PhyB-CreC and PIF6-CreN. The Cre activity is blocked by splitting Cre into an N-terminal half (CreN) and a C-terminal half. By fusing the two halves of Cre with the Arabidopsis thaliana light-inducible binding partners, PhyB and PIF6, the split Cre can become active by red light (660 nm) illumination. Transgenic mice, CreLite, carrying both synthetic genes have been generated. Mouse embryonic fibroblasts (MEFs) derived from CreLite; R26R-YFP mouse embryos successfully turned on the YFP reporter after exposure to red light. Furthermore, microinjection of CreLite mRNAs and its cofactor (phycocyanobilin; PCB) into Zebrabow embryos and red light illumination resulted in Cre activity and the generation of diverse color profiles in various embryonic tissues, including heart, skeletal muscle and epithelium. CreLite is a new tool for precise temporal and spatial control of gene expression in cell culture, ex vivo organ culture, and animal models for developmental biology studies.","createdAt":"2016-10-13T20:55:53.236Z","submittedAt":["2016-10-13T21:29:43.356Z","2016-10-13T21:30:56.609Z","2016-10-13T21:33:20.489Z"],"canGiveATalk":"true","updatedAt":"2016-10-13T21:33:20.490Z"},{"_id":"5801573040b18100036a0009","email":"ping-hung.chen@utsouthwestern.edu","title":"Crosstalk Between CLCb/Dyn1-Mediated Adaptive CME and EGFR Signaling Increases Metastasis.","field":"Translational biology","authors":[{"name":"Ping-Hung Chen","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true},{"name":"Nawal Bendris","role":"General","affiliationSup":[1,null],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true},{"name":"Yi-Jing Hsiao","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Clinical Laboratory Sciences and Medical Biotechnology, National Taiwan University College of Medicine, Taipei, Taiwan"],"validAuthor":true},{"name":"Carlos R Reis","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true},{"name":"Marcel Mettlen","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true},{"name":"Hsuan-Yu Chen","role":"General","affiliationSup":[3],"affiliationOfAuthor":["Institute of Statistical Science, Academia Sinica, Taipei, Taiwan"],"validAuthor":true},{"name":"Sung-Liang Yu","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Clinical Laboratory Sciences and Medical Biotechnology, National Taiwan University College of Medicine, Taipei, Taiwan"],"validAuthor":true},{"name":"Sandra L Schmid","role":"Corresponding","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true}],"affiliations":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX","Department of Clinical Laboratory Sciences and Medical Biotechnology, National Taiwan University College of Medicine, Taipei, Taiwan","Institute of Statistical Science, Academia Sinica, Taipei, Taiwan"],"useAffiliationSup":true,"keywords":"Clathrin-mediated endocytosis, dynamin-1, clathrin light chain, receptor recycling, Akt/GSK3β signaling, non-small cell lung cancer (NSCLC), cell migration, metastasis","absContent":"Signaling receptors are internalized and regulated by clathrin-mediated endocytosis (CME). Two clathrin light chain isoforms, CLCa and CLCb are integral components of the endocytic machinery whose differential functions remain unknown. We report that CLCb is specifically up-regulated in non-small cell lung cancer (NSCLC) cells and associated with poor patient prognosis. Engineered single CLCb-expressing NSCLC cells, as well as ‘switched’ cells that predominantly express CLCb, exhibit increased rates of CME and altered clathrin-coated pit dynamics. This ‘adaptive CME’ resulted from up-regulation of dynamin 1 (Dyn1) and its activation through a positive feedback loop involving enhanced EGF-dependent Akt/GSK3β phosphorylation. CLCb/Dyn1-dependent adaptive CME selectively altered EGFR trafficking, enhanced cell migration in vitro, and increased the metastatic efficiency of NSCLC cells in vivo. We define molecular mechanisms for adaptive CME in cancer cells and a role for the reciprocal crosstalk between signaling and CME in cancer progression.","createdAt":"2016-10-14T21:58:27.204Z","submittedAt":["2016-10-14T22:07:44.161Z","2016-10-14T22:08:17.019Z"],"canGiveATalk":"true","updatedAt":"2016-10-14T22:08:17.021Z"},{"_id":"5802cf559e645c00038970a7","email":"schou@uttyler.edu","title":"Drug-Polymer Interactions Affect Mechanical and Release Behavior of Drug-Eluting Fiber","field":"Bioengineering","authors":[{"name":"Shih-Feng Chou","role":"Presenting","affiliationSup":[1,2],"affiliationOfAuthor":["Department of Mechanical Engineering, University of Texas at Tyler, Tyler, TX","Department of Bioengineering, University of Washington, Seattle, WA"],"validAuthor":true},{"name":"Kim A. Woodrow","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Bioengineering, University of Washington, Seattle, WA"],"validAuthor":true},{"name":"Shih-Feng Chou","role":"Corresponding","affiliationSup":[1,null],"affiliationOfAuthor":["Department of Mechanical Engineering, University of Texas at Tyler, Tyler, TX",""],"validAuthor":true}],"affiliations":["Department of Mechanical Engineering, University of Texas at Tyler, Tyler, TX","Department of Bioengineering, University of Washington, Seattle, WA"],"useAffiliationSup":true,"keywords":"Electrospun nanofibers, Drug Release, Mechanical properties","absContent":"Electrospun drug-eluting fibers have demonstrated advantages in loading efficiency and ability to tune release profiles compared to other therapeutic carriers used for implantable medical devices. In such instance, drug loading may significantly affect mechanical performance and release behavior of fibers. Here, bulk mechanical properties of blend fibers made from polycaprolactone (PCL) and poly(D,L-lactic-_co_-glycolic) acid (PLGA) at various weight ratios and combined with up to 40 wt% of a small molecule water-soluble drug (tenofovir, TFV) were measured to inform drug-polymer interactions at the molecular level. HPLC analysis was used to quantify drug concentration in the release media for up to 10 days.\n\nUniaxial tensile tests were performed on electrospun PCL/PLGA fibers, and the resulting average Young’s moduli and tensile strength of blank fibers showed an gradual increase with increasing PLGA contents in the blend fibers. However, TFV loaded fibers suggested a different trend of mechanical properties from blank fibers, indicating the effect of drug-polymer interactions. In addition, increasing drug loading to 40 wt% decreased the mechanical properties of PCL/PLGA 20:80 fibers due to a strong plasticizing effect from the drug, and consequently led to a burst release at higher drug loading. Dogbone samples collected from the release media at predetermined time points showed significant decreases in average Young’s modulus and tensile strength as compared to the blank fibers. The additional loss in mechanical properties was associated with drug release behaviors in addition to biodegradation of the polymer. Interestingly, TFV release rates increased in pre-stretched fibers. Mechanical assessment on drug partition in PCL/PLGA fibers suggested a higher drug content in the PCL phase than in the PLGA phase. This study contributes significantly to the understanding of drug-polymer interactions in electrospun drug-eluting fibers and provides important property information for implantable therapeutic carriers in future clinical applications.","createdAt":"2016-10-16T00:41:19.902Z","submittedAt":[null,"2016-10-16T01:42:47.274Z"],"updatedAt":"2016-10-16T01:42:52.886Z","canGiveATalk":"true"}];
            // let rawData = [{"_id":"57f168e252f9a10003cd79b0","email":"asdscott@gmail.com","title":"Eg5 Inhibitors Have Contrasting Effects on Microtubule Stability and Metaphase Spindle Integrity","field":"Bioengineering","authors":[{"name":"Geng-Yuan Chen","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Department of Bioengineering, Pennsylvania State University, University Park, PA"],"validAuthor":true},{"name":"You-Jung Kang","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Bioengineering, Pennsylvania State University, University Park, PA"],"validAuthor":true},{"name":"A. Sophia Gayek","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN"],"validAuthor":true},{"name":"Wiphu Youyen","role":"General","affiliationSup":[3],"affiliationOfAuthor":["Department of Physics, Worcester Polytechnic Institute, Worcester, MA"],"validAuthor":true},{"name":"Erkan Tüzel","role":"General","affiliationSup":[3],"affiliationOfAuthor":["Department of Physics, Worcester Polytechnic Institute, Worcester, MA"],"validAuthor":true},{"name":"Ryoma Ohi","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN"],"validAuthor":true},{"name":"William O. Hancock","role":"Corresponding","affiliationSup":[1,null],"affiliationOfAuthor":["Department of Bioengineering, Pennsylvania State University, University Park, PA"],"validAuthor":true}],"affiliations":["Department of Bioengineering, Pennsylvania State University, University Park, PA","Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN","Department of Physics, Worcester Polytechnic Institute, Worcester, MA"],"useAffiliationSup":true,"keywords":"kinesin, microtubule, mitosis, single-molecule enzyme kineitcs","absContent":"The goal was to investigate the mechanisms underlying different Eg5 inhibitors and to assess their effects on Eg5-mediated microtubules (MTs) stabilization.  Antimitotic Eg5 inhibitors were grouped into those that bind Loop-5 and produce an ADP-like state (monastrol, STLC, ispinesib and filanesib) and those that produce a rigor-like state (BRD9876).  To understand their contrasting effects on Eg5 mechanics, inhibitors were analyzed in MT motility assays consisting of different ratios of kinesin-1 and Eg5 motors, in which Eg5 “braking” dominates motility. Loop-5 inhibitors increased MT motility, consistent with the compounds inducing a weak-binding Eg5, whereas BRD9876 slowed gliding, consistent with Eg5 being locked in a \"rigor\". Quantitative stall force analysis indicates that L5 inhibitors reduce Eg5 detachment force whereas BRD9876 generates an Eg5 species with enhanced detachment force. Biochemical and single-molecule assays demonstrated that BRD9876 acts as an ATP/ADP-competitive inhibitor with a KI of 4 nM. Consistent with its recently reported MT polymerase activity _(Chen and Hancock, Nat. Comm. 2015:8160)_ and its dominant two-head-bound strong-binding population _(Chen et al., JBC 2016:M116)_, Eg5 was shown to stabilize MTs against depolymerization. This stabilization was eliminated in monastrol but was enhanced by BRD9876, consistent with strong binding of Eg5 to MTs enhancing MT stability. Finally, in metaphase-arrested RPE-1 cells, STLC promoted spindle collapse, whereas BRD9876 did not. Thus, different Eg5 inhibitors inhibit spindle formation through contrasting mechanisms and rigor inhibitors are predicted to stabilize MTs in cells.","createdAt":"2016-10-02T19:30:10.907Z","submittedAt":["2016-10-02T20:06:57.840Z"],"canGiveATalk":"true"},{"_id":"57fe81d6325e1700036473bc","email":"crjray@gmail.com","title":"Structural Studies and Structure-Guided Engineerings of Epimerization Domains to Reprogram Antibiotic Biosynthetic Pathways","field":"Biochemistry","authors":[{"name":"Wei-Hung Chen","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Department of Chemistry, University of Florida, Gainesville, FL"],"validAuthor":true},{"name":"Naga Sandhya Guntaka","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Chemistry, University of Florida, Gainesville, FL"],"validAuthor":true},{"name":"Steven D. Bruner","role":"Corresponding","affiliationSup":[1],"affiliationOfAuthor":["Department of Chemistry, University of Florida, Gainesville, FL"],"validAuthor":true}],"affiliations":["Department of Chemistry, University of Florida, Gainesville, FL"],"useAffiliationSup":false,"keywords":"nonribosomal peptide synthetase, epimerase, and antibiotics biosynthesis pathway","absContent":"Nonribosomal peptide synthetases are large, complex multi-domain enzymes responsible for the biosynthesis of a wide range of peptidyl natural products. The addition of further chemical modifications contribute significantly to the structural diversity of nonribosomal peptides, adding functional diversity and _in vivo_ stability. Notably, the epimerization (E) domain in NRPS assembly lines is a common and important modification for bioactive nonribosomal peptides.\n\nDespite the previous biochemical and structural studies, several questions related to E domain substrate recognition and the detailed chemical mechanism remain to be addressed. In addition, general questions related to NRPS domain orientation and the flux of Ppant-tethered intermediates through the megasynthetase are unclear. The issues of structure and mechanism have direct relevance to engineer NRPSs through domain manipulations and rearrangements to produce novel compounds, although studies have shown success, generally applicable approaches are not known. \n\nIn this study, we present the structure of the epimerization domain/peptidyl carrier protein didomain construct from the first module of the gramicidin S synthetase. From the structures, novel details related to the interactions between catalytic residues/the Ppant prosthetic group and the interface region within linker-domain and domain-domain are apparent. Additionally, the information revealed within structures provides insights into domain-domain recognition, substrate delivery during the assembly line process and guides us to carry out a structural based engineering resulting in a relaxed substrate specificity. Finally, the structures supply information into for the structural organization of homologous condensation domains, canonical players in all synthetase modules.","createdAt":"2016-10-12T17:52:33.270Z","submittedAt":["2016-10-12T18:32:56.150Z","2016-10-12T18:34:54.459Z","2016-10-12T18:36:59.061Z"],"canGiveATalk":"true","updatedAt":"2016-10-12T18:36:59.063Z"},{"_id":"57fffcbd6856f40003948d10","email":"yenshuoting@gmail.com","title":"CreLite: A Red Light-Inducible Cre Tool for Mouse and Zebrafish","field":"Developmental biology","authors":[{"name":"Shuo-Ting Yen","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Program in Developmental Biology, Baylor College of Medicine, Houston, TX"],"validAuthor":true},{"name":"Jian-Min Deng","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Genetics, University of Texas MD Anderson Cancer Center, Houston, TX"],"validAuthor":true},{"name":"George T. Einsenhoffer","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Genetics, University of Texas MD Anderson Cancer Center, Houston, TX"],"validAuthor":true},{"name":"Richard R. Behringer","role":"Corresponding","affiliationSup":[2],"affiliationOfAuthor":["Department of Genetics, University of Texas MD Anderson Cancer Center, Houston, TX"],"validAuthor":true}],"affiliations":["Program in Developmental Biology, Baylor College of Medicine, Houston, TX","Department of Genetics, University of Texas MD Anderson Cancer Center, Houston, TX"],"useAffiliationSup":true,"keywords":"Optogenetics, Cre recombinase, Phytochrome B, phycocyanobilin","absContent":"Precise manipulation of gene expression with spatial and temporal control has been an important goal for developmental biology studies. Currently, the Cre-loxP binary system allows gene expression control using known ubiquitous or tissue-specific regulatory elements, which limits the possibilities for the control of gene expression at desired times and places. Recently, light-inducible proteins have become a powerful tool for genetic manipulations and confer precise control of various biological systems. A blue light-inducible Cre system has been engineered and applied in fly and a cell culture system. However, the low tissue penetrance and high photo-toxicity of blue light may hinder further applications for live, thick tissue or animals. Here, we present an engineered red light-inducible Cre system, CreLite, for potential use in both mouse cell culture and zebrafish embryos. The red light-inducible Cre system consists of two fusion proteins, PhyB-CreC and PIF6-CreN. The Cre activity is blocked by splitting Cre into an N-terminal half (CreN) and a C-terminal half. By fusing the two halves of Cre with the Arabidopsis thaliana light-inducible binding partners, PhyB and PIF6, the split Cre can become active by red light (660 nm) illumination. Transgenic mice, CreLite, carrying both synthetic genes have been generated. Mouse embryonic fibroblasts (MEFs) derived from CreLite; R26R-YFP mouse embryos successfully turned on the YFP reporter after exposure to red light. Furthermore, microinjection of CreLite mRNAs and its cofactor (phycocyanobilin; PCB) into Zebrabow embryos and red light illumination resulted in Cre activity and the generation of diverse color profiles in various embryonic tissues, including heart, skeletal muscle and epithelium. CreLite is a new tool for precise temporal and spatial control of gene expression in cell culture, ex vivo organ culture, and animal models for developmental biology studies.","createdAt":"2016-10-13T20:55:53.236Z","submittedAt":["2016-10-13T21:29:43.356Z","2016-10-13T21:30:56.609Z","2016-10-13T21:33:20.489Z"],"canGiveATalk":"true","updatedAt":"2016-10-13T21:33:20.490Z"},{"_id":"5801573040b18100036a0009","email":"ping-hung.chen@utsouthwestern.edu","title":"Crosstalk Between CLCb/Dyn1-Mediated Adaptive CME and EGFR Signaling Increases Metastasis.","field":"Translational biology","authors":[{"name":"Ping-Hung Chen","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true},{"name":"Nawal Bendris","role":"General","affiliationSup":[1,null],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true},{"name":"Yi-Jing Hsiao","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Clinical Laboratory Sciences and Medical Biotechnology, National Taiwan University College of Medicine, Taipei, Taiwan"],"validAuthor":true},{"name":"Carlos R Reis","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true},{"name":"Marcel Mettlen","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true},{"name":"Hsuan-Yu Chen","role":"General","affiliationSup":[3],"affiliationOfAuthor":["Institute of Statistical Science, Academia Sinica, Taipei, Taiwan"],"validAuthor":true},{"name":"Sung-Liang Yu","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Clinical Laboratory Sciences and Medical Biotechnology, National Taiwan University College of Medicine, Taipei, Taiwan"],"validAuthor":true},{"name":"Sandra L Schmid","role":"Corresponding","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX"],"validAuthor":true}],"affiliations":["Department of Cell biology, University of Texas Southwestern Medical Center, Dallas, TX","Department of Clinical Laboratory Sciences and Medical Biotechnology, National Taiwan University College of Medicine, Taipei, Taiwan","Institute of Statistical Science, Academia Sinica, Taipei, Taiwan"],"useAffiliationSup":true,"keywords":"Clathrin-mediated endocytosis, dynamin-1, clathrin light chain, receptor recycling, Akt/GSK3β signaling, non-small cell lung cancer (NSCLC), cell migration, metastasis","absContent":"Signaling receptors are internalized and regulated by clathrin-mediated endocytosis (CME). Two clathrin light chain isoforms, CLCa and CLCb are integral components of the endocytic machinery whose differential functions remain unknown. We report that CLCb is specifically up-regulated in non-small cell lung cancer (NSCLC) cells and associated with poor patient prognosis. Engineered single CLCb-expressing NSCLC cells, as well as ‘switched’ cells that predominantly express CLCb, exhibit increased rates of CME and altered clathrin-coated pit dynamics. This ‘adaptive CME’ resulted from up-regulation of dynamin 1 (Dyn1) and its activation through a positive feedback loop involving enhanced EGF-dependent Akt/GSK3β phosphorylation. CLCb/Dyn1-dependent adaptive CME selectively altered EGFR trafficking, enhanced cell migration in vitro, and increased the metastatic efficiency of NSCLC cells in vivo. We define molecular mechanisms for adaptive CME in cancer cells and a role for the reciprocal crosstalk between signaling and CME in cancer progression.","createdAt":"2016-10-14T21:58:27.204Z","submittedAt":["2016-10-14T22:07:44.161Z","2016-10-14T22:08:17.019Z"],"canGiveATalk":"true","updatedAt":"2016-10-14T22:08:17.021Z"},{"_id":"5802cf559e645c00038970a7","email":"schou@uttyler.edu","title":"Drug-Polymer Interactions Affect Mechanical and Release Behavior of Drug-Eluting Fiber","field":"Bioengineering","authors":[{"name":"Shih-Feng Chou","role":"Presenting","affiliationSup":[1,2],"affiliationOfAuthor":["Department of Mechanical Engineering, University of Texas at Tyler, Tyler, TX","Department of Bioengineering, University of Washington, Seattle, WA"],"validAuthor":true},{"name":"Kim A. Woodrow","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Bioengineering, University of Washington, Seattle, WA"],"validAuthor":true},{"name":"Shih-Feng Chou","role":"Corresponding","affiliationSup":[1,null],"affiliationOfAuthor":["Department of Mechanical Engineering, University of Texas at Tyler, Tyler, TX",""],"validAuthor":true}],"affiliations":["Department of Mechanical Engineering, University of Texas at Tyler, Tyler, TX","Department of Bioengineering, University of Washington, Seattle, WA"],"useAffiliationSup":true,"keywords":"Electrospun nanofibers, Drug Release, Mechanical properties","absContent":"Electrospun drug-eluting fibers have demonstrated advantages in loading efficiency and ability to tune release profiles compared to other therapeutic carriers used for implantable medical devices. In such instance, drug loading may significantly affect mechanical performance and release behavior of fibers. Here, bulk mechanical properties of blend fibers made from polycaprolactone (PCL) and poly(D,L-lactic-_co_-glycolic) acid (PLGA) at various weight ratios and combined with up to 40 wt% of a small molecule water-soluble drug (tenofovir, TFV) were measured to inform drug-polymer interactions at the molecular level. HPLC analysis was used to quantify drug concentration in the release media for up to 10 days.\n\nUniaxial tensile tests were performed on electrospun PCL/PLGA fibers, and the resulting average Young’s moduli and tensile strength of blank fibers showed an gradual increase with increasing PLGA contents in the blend fibers. However, TFV loaded fibers suggested a different trend of mechanical properties from blank fibers, indicating the effect of drug-polymer interactions. In addition, increasing drug loading to 40 wt% decreased the mechanical properties of PCL/PLGA 20:80 fibers due to a strong plasticizing effect from the drug, and consequently led to a burst release at higher drug loading. Dogbone samples collected from the release media at predetermined time points showed significant decreases in average Young’s modulus and tensile strength as compared to the blank fibers. The additional loss in mechanical properties was associated with drug release behaviors in addition to biodegradation of the polymer. Interestingly, TFV release rates increased in pre-stretched fibers. Mechanical assessment on drug partition in PCL/PLGA fibers suggested a higher drug content in the PCL phase than in the PLGA phase. This study contributes significantly to the understanding of drug-polymer interactions in electrospun drug-eluting fibers and provides important property information for implantable therapeutic carriers in future clinical applications.","createdAt":"2016-10-16T00:41:19.902Z","submittedAt":[null,"2016-10-16T01:42:47.274Z"],"updatedAt":"2016-10-16T01:42:52.886Z","canGiveATalk":"true"},{"_id":"5803cb9bb95acb000374f819","email":"atku@bcm.edu","title":"A Noncanonical Role of Tcf3 in Skin Tumorigenesis","field":"Translational biology","authors":[{"name":"Amy T. Ku","role":"Presenting","affiliationSup":[1,2],"affiliationOfAuthor":["Interdepartmental Program in Translational Biology and Molecular Medicine, Baylor College of Medicine, Houston, TX","Center for Cell and Gene Therapy, Baylor College of Medicine, Houston, TX"],"validAuthor":true},{"name":"Hoang Nguyen","role":"Corresponding","affiliationSup":[3,1,2],"affiliationOfAuthor":["Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston, TX","Interdepartmental Program in Translational Biology and Molecular Medicine, Baylor College of Medicine, Houston, TX","Center for Cell and Gene Therapy, Baylor College of Medicine, Houston, TX"],"validAuthor":true}],"affiliations":["Interdepartmental Program in Translational Biology and Molecular Medicine, Baylor College of Medicine, Houston, TX","Center for Cell and Gene Therapy, Baylor College of Medicine, Houston, TX","Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston, TX"],"useAffiliationSup":true,"keywords":"skin cancer, Wnt signaling, TCF, β-catenin, lipocalin 2","absContent":"Non-melanoma skin cancer (NMSC) is the most common cancer in the Westernized countries. Although canonical Wnt signaling has served as a promising therapeutic target, its efficacy in NMSC has not yet defined. Tcf3 (or Tcf7l1), a Lef/Tcf gene family member, is a β-catenin cofactor in canonical Wnt signaling. Our previous studies present Tcf3 as a tumor promoter in chemically-induced murine model of skin squamous cell carcinoma (SCC). Understanding how Tcf3 contributes to skin tumorigenesis may therefore benefit NMSC treatments. In this study, we demonstrate TCF3 overexpression increases the tumorigenic potential of skin SCC cells using limiting-dilution cell transplantation assay. Meanwhile, knocking down both TCF3 and TCF4, the functional paralogue to TCF3, decreases the growth of xenografted tumors. Follow-up immunostaining and in vitro assays conclude TCF3-altered cellular pathways, cell proliferation and cell migration, converge into sustained tumor growth. To evaluate TCF3's functionality, we employ separation-of-function mutants. Interestingly, TCF3 deletion mutant without β-catenin binding capacity (TCF3ΔN) retains pro-tumorigenic activities, whereas other deletion mutants exhibit diminished activities. These findings suggest TCF3 utilizes alternate pathways to canonical Wnt/β-catenin signaling during tumor development. We further identify a secreted protein lipocalin 2 (LCN2) as downstream effector to TCF3 in tumor growth. In-depth studies of TCF3-altered transcriptional landscape implicate non-canonical Wnt/Calcium pathway and inflammation may contribute to TCF3-promoted skin tumorigenesis. Further studies will warrant the identification of biomarkers and therapeutic targets in skin SCC.","createdAt":"2016-10-16T18:43:30.833Z","submittedAt":[null,"2016-10-17T01:07:21.025Z"],"updatedAt":"2016-10-17T01:07:25.193Z","canGiveATalk":"true"},{"_id":"58042dbab95acb000374f81d","email":"cltien@uab.edu","title":"Heterochromatin Protein 1β Regulates Neural and Neural Crest Development via\nRepression of The Pluripotency-Associated\nGene Oct25","field":"Developmental biology","authors":[{"name":"Chih-Liang Tien","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell, Developmental and Integrative Biology, University of Alabama at Birmingham, Birmingham, AL"],"validAuthor":true},{"name":"Chenbei Chang","role":"Corresponding","affiliationSup":[1],"affiliationOfAuthor":["Department of Cell, Developmental and Integrative Biology, University of Alabama at Birmingham, Birmingham, AL"],"validAuthor":true}],"affiliations":["Department of Cell, Developmental and Integrative Biology, University of Alabama at Birmingham, Birmingham, AL"],"useAffiliationSup":false,"keywords":"Epigenetics, Heterochromatin protein 1, Oct25, Xenopus","absContent":"Formation of the vertebrate nervous system starts at neurulation, whereby neural and neural crest cells are induced in the dorsal ectoderm and give rise to central and periphery nervous systems respectively during later embryogenesis. Multiple growth factor signals and transcription factor networks have been implicated in this process, but how epigenetic mechanisms regulate neural and neural crest development is less understood. In this study, we examined the function of heterochromatin protein 1 beta (HP1β), a histone H3 lysine 9 trimethylation (H3K9me3) binding protein involved in modulation of chromatin structure, in early development of the nervous system in Xenopus. We found that all three vertebrate HP1 paralogs, HP1α, β, and γ, were expressed in the neural and neural crest regions during neurulation. Antisense morpholino (MO)-mediated knockdown of HP1β and HP1γ, but not HP1α, reduced the expression of the late neural marker NRP1 and the neural crest markers Sox9 and Snail2, but did not affect the early neural marker Sox2 or the neural plate border marker AP2α. As the embryos were most sensitive to the level of HP1β-MO, we focused our studies on this protein. We showed that HP1β but not HP1α or HP1γ, rescued the defects in HP1β morphant, implying that HP1 paralogs had distinct activities. As early genes induced in the dorsal ectoderm were not affected by HP1β-MO, but late neural and neural crest markers were reduced, we hypothesized that HP1β might regulate switch between precursor and more differentiated cell types. In agreement with this idea, we observed expansion of Oct25, a Xenopus Oct4 homolog associated with pluripotency, in HP1β morphant. Overexpression of Oct25 phenocopied HP1β-MO in regulation of NRP1, Sox9 and Snail2; and knockdown of Oct25 partially rescued the expression of these markers in HP1β morphant. Taken together, our data indicate that HP1β promotes neural and neural crest differentiation at least partially via repression of the pluripotency-associated gene Oct25.","createdAt":"2016-10-17T01:18:10.251Z","submittedAt":["2016-10-17T01:47:36.423Z"],"canGiveATalk":"true"},{"_id":"5804335cb95acb000374f81e","email":"ylliu@utexas.edu","title":"Segmentation Analysis for Trajectories Acquired by The TSUNAMI Tracking Microscope: An Application to EGFR Trafficking","field":"Biophysics","authors":[{"name":"Yen-Liang Liu","role":"Presenting","affiliationSup":[1],"affiliationOfAuthor":["Department of Biomedical Engineering, The University of Texas at Austin, Austin, TX, USA"],"validAuthor":true},{"name":"Evan P. Perillo","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Biomedical Engineering, The University of Texas at Austin, Austin, TX, USA"],"validAuthor":true},{"name":"Cong Liu","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Biomedical Engineering, The University of Texas at Austin, Austin, TX, USA"],"validAuthor":true},{"name":"Peter Yu","role":"General","affiliationSup":[1],"affiliationOfAuthor":["Department of Biomedical Engineering, The University of Texas at Austin, Austin, TX, USA"],"validAuthor":true},{"name":"Chao-Kai Chou","role":"General","affiliationSup":[2],"affiliationOfAuthor":["Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA"],"validAuthor":true},{"name":"Mien-Chie Hung","role":"General","affiliationSup":[2,3],"affiliationOfAuthor":["Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA","Center for Molecular Medicine and Graduate Institute of Cancer Biology, China Medical University, Taichung, Taiwan"],"validAuthor":true},{"name":"Hsin-Chih (Tim) Yeh","role":"Corresponding","affiliationSup":[1],"affiliationOfAuthor":["Department of Biomedical Engineering, The University of Texas at Austin, Austin, TX, USA"],"validAuthor":true}],"affiliations":["Department of Biomedical Engineering, The University of Texas at Austin, Austin, TX, USA","Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA","Center for Molecular Medicine and Graduate Institute of Cancer Biology, China Medical University, Taichung, Taiwan"],"useAffiliationSup":true,"keywords":"single-particle tracking, epidermal growth factor receptor, endocytosis.","absContent":"Whereas important discoveries made by single-particle tracking have changed our view of the plasma membrane organization and motor protein dynamics in the past three decades, experimental studies of intracellular processes using single-particle tracking are rather scarce due to the lack of 3D tracking capacity. Here we use a newly developed 3D single-particle tracking method termed TSUNAMI (Tracking of Single particles Using Nonlinear And Multiplexed Illumination) to investigate EGFR trafficking dynamics in live cells at 16/43 nm (xy/z) spatial resolution, with track duration ranging from 2 to 10 minutes and vertical tracking depth up to tens of microns. To analyze the long 3D trajectories generated by the TSUNAMI microscope, a trajectory analysis algorithm is developed, which reaches 81% segment classification accuracy in control experiments (termed simulated movement experiments). When analyzing 95 EGF-stimulated EGFR trajectories acquired in live skin cancer cells, we find that these trajectories can be separated into three groups – immobilization (24.2%), membrane diffusion only (51.6%), and transport from membrane to cytoplasm (24.2%). When EGFRs are membrane bound, they show an interchange of Brownian diffusion and confined diffusion. When EGFRs are internalized, transitions from confined diffusion to directed diffusion and from directed diffusion back to confined diffusion are clearly seen. This observation agrees well with the model of clathrin-mediated endocytosis.","createdAt":"2016-10-17T01:43:08.085Z","submittedAt":["2016-10-17T02:11:38.485Z"]}];
            // let rawData = [];
            this.FormApi.getAbstracts()
            .then(
                (res)=>{
                    let rawData = res.data;
                    
                    for (let i=0; i<rawData.length; i++) {
                        let authorNameArray = rawData[i].authors[0].name.split(' ');
                        rawData[i].fileId = authorNameArray[authorNameArray.length-1] + "_" + authorNameArray[0];
                        rawData[i].fieldNoSpace = rawData[i].field.replace(/\s/g, '-');
                        this.selectAllAbs.push(rawData[i].fieldNoSpace+'-'+rawData[i].fileId);
                    }
                    // console.log(this.selectAllAbs);
                    this.abstracts =  rawData;
                    
                },
                ()=>{
                    console.log('error: getAbstracts');
                }
            )
            
            // for (let i=0; i<rawData.length; i++) {
            //     let authorNameArray = rawData[i].authors[0].name.split(' ');
            //     rawData[i].fileId = authorNameArray[authorNameArray.length-1] + "_" + authorNameArray[0];
            //     rawData[i].fieldNoSpace = rawData[i].field.replace(/\s/g, '-');
            //     this.selectAllAbs.push(rawData[i].fieldNoSpace+'-'+rawData[i].fileId);
            // }
            // console.log(this.selectAllAbs);
            // return rawData;
            
        }
        replaceSpace(str) {
            return str.replace(/\s/g, '-');
        }
        toggleSelectAbs(email, selectedList) {
            let idx = selectedList.indexOf(email);
            if (idx > -1) {
                selectedList.splice(idx, 1);
            } else {
                selectedList.push(email);
            }
        }
        absSelected(item, list) {
            return list.indexOf(item) > -1;
        }
        toggleSelectAll() {
            if (this.selectedAbs.length == this.abstracts.length) {
                this.selectedAbs = [];
            } else {
                this.selectedAbs = this.selectAllAbs;
            }
        }
        allSelected() {
            return this.selectedAbs.length == this.abstracts.length;
        }
        generateFile(selectedAbs) {
            for (let i=0; i<selectedAbs.length; i++) {
                let label = document.querySelector(
                                '#label_' + selectedAbs[i] + ' span'
                            ).innerHTML.trim();
                if (label.length === 1) {
                    label = "0" + label;
                }
                console.log(label);
                let absElId = '#abs_' + selectedAbs[i];
                saveAs(this.convertHtmlToDocx(absElId), label + '_' + selectedAbs[i] + '.docx');
            }
        }
        
        convertHtmlToDocx(elementId) {
            // $document
            // this.$document[0].getElementById();
            var el = document.querySelector(elementId);
            // some custom code for formatting
            var indexForDeletion = el.innerHTML.lastIndexOf("<br>");
            var strForDocx = el.innerHTML.substring(0, indexForDeletion) + 
                             el.innerHTML.substring(indexForDeletion+4);
            var content = '<!DOCTYPE html>'+ `
            <style>
            body {
                font-family: Arial;
                font-size: 14.5px;
                line-height: 25px;
            }
            
            .abs_title {
                font-weight: 600;
                margin-top: 0px;
                margin-bottom: 29px;
                font-size: 14.5px;
            }
            .wrapper-author {
                font-weight: 400;
                margin-bottom: -290px;
            }
            .wrapper-affil {
                font-size: 12.75px;
                margin-top: 0px;
                margin-bottom: 29px;
            }
            .wrapper-keywords {
                font-size: 13px;
                font-style: italic;
                font-weight: 600;
                margin-top: 0px;
                margin-bottom: 14.5px;
            }
            .wrapper-abs-content {
                text-align: justify;
            }
            </style>
            ` + '<body>' + strForDocx + '</body>';
            var converted = htmlDocx.asBlob(content);
            return converted;
        }
    }
};
export default adminComponent;
